functional-tests/http/src/test/java/org/mule/test/http/functional/requester/HttpRequestMaxConnectionsTestCase.java

```diff
@@ -58,6 +57,7 @@ public void maxConnections() throws Exception {
   private Thread processAsynchronously(final Flow flow) {
     Thread thread = new Thread(() -> {
       try {
+        System.out.println("Sent one request");
```

Comment:
Remove the print or replace it with a logger

Reply:
It's not needed, removed